airbyte-integrations/connectors/destination-bigquery/Dockerfile

```diff
@@ -17,5 +17,5 @@ ENV ENABLE_SENTRY true
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=1.2.9
+LABEL io.airbyte.version=1.2.10
```

Comment:
I should introduce a major version bump as this will likely break normalization on old OSS versions